.github/workflows/auto-reqs-update-pr.yml

```diff
@@ -3,7 +3,7 @@ on:
   workflow_dispatch
 jobs:
   auto-pull-request:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04
```

Comment:
just curious: is this change to be explicit instead of implicit (that I would very much approve of), or there is a factual change?

Reply:
There is no actual change. I just thought it is nicer to have this pinned and to be uniform everywhere.